### ISSUE TITLE
Info: Status not needed in CustomerFAQZoom (and public?)

### ISSUE THREAD

**Issue by StefanRother-OTOBO** [RotherOSS/FAQ#4]
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/56070449/99248924-8bc9b180-2809-11eb-8d7c-84783c0ccfb6.png">

**Comment by wollmers** [comment]
Sure, the whole FAQ Information Box looks more technical.

The only value for a customer user could be, to know if the FAQ is visible on the public interface or not.

![Bildschirmfoto 2020-11-16 um 13 33 04](https://user-images.githubusercontent.com/1275557/99253349-e286b980-2810-11eb-90f0-3864ec648397.png)

**Comment by StefanRother-OTOBO** [comment]
No, a CustomerUser does not need the information whether the article is public or not. These entries are also only intended as a reminder. We discuss them later and decide what we do.

But while we are on the subject. What I also miss is the information which FAQ article is assigned to which service in AgentFAQZoom. And the possibility to search for FAQ articles of a service is missing in AgentFAQSearch. Furthermore the documentation in the package is not correct. There is still talk about OTRS and the new function is not described.

**Comment by wollmers** [comment]
Opened issue #5 for Agent area

**Comment by wollmers** [comment]
Opened issue #6 for docs

**Comment by stefanhaerter** [comment]
Putting this in the OTOBO Issues project for planning.
